src/BusinessLayer/TelemetryReporting.h

```diff
@@ -31,8 +39,12 @@ class TelemetryReporting
 
    void writeFloatIntoData(unsigned char* data, int index, const float& value);
    void sendData(const unsigned char* data, int length);

```

Comment:
writeFloatIntoData has been renamed to writeFloatIntoArray to better suit what the function does.
void sendData has been removed from the header